src/views/Home/Home.test.jsx

```diff
@@ -11,5 +13,6 @@ const user = {
 }
 
 test('Should render the user profile', () => {
-
+  render(<Profile user={user} />)
 })
+// this passed but I am not sure why. Alex explained it to be but I don't see why it passed. I rendered the Profile component and set user to user information. What would be a best practice for this example??
```

Comment:
Here we will want to test for name, motto, the interests heading, avatar, header image, and a list of userLikes to make this test complete.

Reply:
Thank you! That makes way more sense in actually testing something.